docs/source/user-manual/index.md

```diff
@@ -0,0 +1,19 @@
+---
+myst:
+  html_meta:
+    "description": "This is the end-User documentation for Plone 6 Volto frontend."
+    "property=og:description": "Volto is the React-based frontend for the Plone CMS. It is the default UI for the Plone 6 release."
+    "property=og:title": "Frontend"
+    "keywords": "Volto, Plone, frontend, React, User manual"
+---
+
+# User Manual
+
+This is the end-User documentation for Plone 6 Volto frontend. You can also directly navigate to the document of the relevant context from Volto frontend by clicking the help `(?)` icon.
+
```

Comment:
Let's add a todo for future PRs. Once it is merged, it will help guide authors.

````suggestion
```{todo}
The Plone 5.2 documentation for Classic UI is well structured, and should be emulated as the model for the Volto User Manual.

- [Working with Content](https://docs.plone.org/working-with-content/index.html)

Additional sources of content may be used to fill out the Volto User Manual.
Note that the audience for these sources may be a Developer, not an Editor, and therefore might need to be retargeted.

- [Volto Hands-On](https://training.plone.org/voltohandson/index.html)
- [Plone 6 Demo](https://6.demo.plone.org/) for screen shots and videos.

```
````